src/lib/evas/canvas/evas_main.c

```diff
@@ -1,5 +1,6 @@
 #define EFL_INPUT_EVENT_PROTECTED
 
+#include "evas_api.h"
```

Comment:
If `evas_private` is not including it already, we have a problem houston.

Reply:
I will have a lock to see if we can remove this line.

Reply:
[yup](https://github.com/expertisesolutions/efl/runs/864992329), I'll remove this.